### PR TITLE
[DEV-2486] looksrare list for sale (Part 2)

### PR DESCRIPTION
**Pull request by maximgeerinck**
# Description

- Add listing on looksrare (Boilerplate)

### CODE REVIEW

File: src/nft/ERC721NFT.ts [hunk 1]
```diff
@@ -322,17 +327,52 @@ export class ERC721NFT extends AbstractEvmNFT {
...
+      const now = Math.floor(Date.now() / 1000);
+      await instance.listForSale(
+        {
+          tokenId: this._item.tokenId,
+          collection: this._item.contractAddress,
+          strategy: StrategyStandardSaleForFixedPrice[1],
+          currency: Common.Addresses.Weth[1],
+          signer: this.refinable.accountAddress,
+          isOrderAsk: true, // side === "sell"
+          nonce: "0",
```
Comment (mihaiberq) [inline src/nft/ERC721NFT.ts:363]: is this normally hardcoded? shouldnt it come from account nonce or something?

File: src/platform/LooksrarePlatform.ts [hunk 2]
```diff
@@ -35,18 +51,76 @@ export class LooksrarePlatform extends AbstractPlatform {
...
+
+    // create
+    const response = await this.refinable.graphqlClient.request<
+      string,
+      MutationLooksrareListForSaleArgs
+    >(LOOKSRARE_LIST_FOR_SALE, {
+      input: { ...order.params, signature },
+    });
+
+    console.log({ response });
```
Comment (mihaiberq) [inline src/platform/LooksrarePlatform.ts:107]: log here

File: src/nft/ERC721NFT.ts (hunk 1 above)
Comment (mihaiberq) [reply src/nft/ERC721NFT.ts:363]: i just noticed the `getNonce` function inside the LooksRare class. should that one be used instead?

**Review by mihaiberq — CHANGES_REQUESTED** [review]
looks good, can approve as soon as things get clearer with the nonce

File: src/nft/ERC721NFT.ts (hunk 1 above)
Comment (maximgeerinck) [reply src/nft/ERC721NFT.ts:365]: it's not live yet, will do this in the next part

File: src/platform/LooksrarePlatform.ts (hunk 2 above)
Comment (maximgeerinck) [reply src/platform/LooksrarePlatform.ts]: removed

**Review by mihaiberq — APPROVED** [review]

**Pull request merged**